### ISSUE TITLE
Structure of a new training data set

### ISSUE THREAD

**Issue by DanMenCas** [NVlabs/stylegan2-ada#117]
Hi,

How i can create a new data set for use StyleGAN, what should be the structure of this data set?, where i can put the test/validation set and how i should label the images.

Thanks!

**Comment by jasuriy** [comment]
@DanMenCas hi 
did you resolve your issue or found answer to your question ?

**Comment by a-ceron** [comment]
Hi, here are some commands that show how you can create a compressed file to use as a dataset:


```
python dataset_tool.py create_from_images ~/datasets/custom ~/custom-images
python dataset_tool.py display ~/datasets/custom
```
Then, to train, just use:

```

python train.py --outdir=~/training-runs --gpus=1 --data=~/datasets/custom --dry-run
python train.py --outdir=~/training-runs --gpus=1 --data=~/datasets/custom
```
This worked fine for me. I had to modify a few lines, but nothing too complicated.

Also, why do you need test/validation? If you want to use IS or FID, there are some files that show how to pass those datasets, but for training, the previous commands should work.
